README.md

```diff
@@ -15,7 +15,10 @@ AutoGluon automates machine learning tasks enabling you to easily achieve strong
 ## Example
 
 ```python
-# First install package from terminal:  pip install mxnet autogluon
+# First install package from terminal:
+# python3 -m pip install --upgrade pip
+# python3 -m pip install --upgrade "mxnet<2.0.0"
+# python3 -m pip install autogluon
```

Comment:
put these all on the same line?

Reply:
Can't because mxnet must be installed before autogluon, and pip must be updated to get mxnet 1.7, so they need to be separate lines.